src/process/clearHidden.ts

```diff
@@ -20,7 +21,14 @@ export const clearHiddenProcess: ProcessorFnSync<ClearHiddenScope> = (context) =
     if (!scope.clearHidden) {
         scope.clearHidden = {};
     }
-    if (component.hidden && value !== undefined && (!component.hasOwnProperty('clearOnHide') || component.clearOnHide)) {
+    const conditionallyHidden = (scope as ConditionsScope).conditionals?.find((cond) => {
```

Comment:
This is fine for now, but this type of type assertion makes me think we should just type the entire scope object as it is _at runtime_ rather than kind of shifting the types around